.github/ISSUE_TEMPLATE/packaging_tutorial.yml

```
@@ -0,0 +1,79 @@
+name: Packaging tutorial problem
+description: Request help for a problem following a packaging tutorial
+title: Trouble with a packaging tutorial
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for providing feedback on Python packaging!
+
+      To help us help you, please fill out as much of the following as you can. If a question is not relevant, feel free to skip it.
```

Given the number of required questions, this seems like a tease. 😉 

```suggestion
      To help us help you, please fill out as much of the following as you can. 
```